frontend/src/pages/modelRegistrySettings/ModelRegistriesPermissions.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  ClipboardCopy,
+  Tab,
+  TabContent,
+  TabContentBody,
+  Tabs,
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router';
+import { KnownLabels, RoleBindingKind } from '~/k8sTypes';
+import { useGroups } from '~/api';
+import RoleBindingPermissions from '~/concepts/roleBinding/RoleBindingPermissions';
+import { useContextResourceData } from '~/utilities/useContextResourceData';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import { MODEL_REGISTRY_DEFAULT_NAMESPACE } from '~/concepts/modelRegistry/const';
+import { SupportedArea } from '~/concepts/areas';
+import { RoleBindingPermissionsRoleType } from '~/concepts/roleBinding/types';
+import useRoleBindings from './useRoleBindings';
+
+const ModelRegistriesManagePermissions: React.FC = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState('users');
+  const [groups] = useGroups();
+  const roleBindings = useContextResourceData<RoleBindingKind>(useRoleBindings());
+  const { mrName } = useParams();
```

Comment:
This page should probably redirect back to `/modelRegistrySettings` if the `mrName` doesn't match an existing modelRegistry (as part of 404 handling). You could do this by fetching modelRegistries here with `useModelRegistriesBackend` and then if that fetch is done and there isn't a MR matching `mrName`, return a `<Navigate to="/modelRegistrySettings" replace />`. or maybe you could just do that if `filteredRoleBindings.length === 0` since in theory there's always the RB for the default group.

Reply:
If you prefer we can also open a followup bug for this, but we just need to make sure the page doesn't crash if the MR with `mrName` doesn't exist.